funding/funding brief

```diff
@@ -0,0 +1,14 @@
+Berichtgeving over het veranderende klimaat is doorgaans pessimistisch en ingewikkeld. Dit zorgt ervoor dat lezers een aversie ervaren jegens het onderwerp. "Het kost mij teveel om het uit te zoeken en wat ik doe heeft toch niet zo'n groot effect". Het maatschappelijk pessimisme heeft niet alleen een effect op het gedrag van individuen maar  indirect ook op dat van overheden en organisaties.
+
+Dit project wil lezers enthousiast maken voor in het onderwerp door wekelijks in zo'n simpel mogelijke taal een overzicht te geven van recente gebeurtenissen. Wij verwachten dat het aanbieden van een wekelijkse portie positiviteit ervoor zal zorgen dat lezers meer zullen communiceren over het onderwerp en dat dit een verandering van het gedrag zal veroorzaken.
+
+De wekelijkse nieuwsbrief zal bestaan uit:
+- Een diepgaand journalistiek stuk over een bepaalde innovatie, regeling, sociale verandering of investering
+- Samenvattingen van relevante uitstekende journalistieke stukken van externe partijen
+- Een update over de stand van zaken van een eerder besproken onderwerp
```

Comment:
Misschien duidelijkheid toevoegen? Heeft het gewerkt? Waar is de technologie/wet/regelgeving nu in het proces?